addons/website_sale/controllers/cart.py

```diff
@@ -326,26 +333,25 @@ def _get_cart_notification_information(self, order, line_ids):
                     'quantity': float
                     'name': str
                     'description': str
-                    'line_price_total': float
+                    'added_qty_price_total': float
                 }],
             }
         """
-        lines = order.order_line.filtered(lambda line: line.id in line_ids)
+        lines = order.order_line.filtered(lambda line: line.id in set(added_qty_per_line))
         if not lines:
             return {}
 
-        show_tax = order.website_id.show_line_subtotals_tax_selection == 'tax_included'
         return {
             'currency_id': order.currency_id.id,
             'lines': [
                 { # For the cart_notification
                     'id': line.id,
                     'image_url': order.website_id.image_url(line.product_id, 'image_128'),
-                    'quantity': line._get_displayed_quantity(),
+                    'added_qty': added_qty_per_line[line.id],
```

Comment:
I'd not change this variable as it's still the quantity that would display so no need to clarify here
```suggestion
                    'quantity': added_qty_per_line[line.id],
```